packages/flame/lib/src/camera/viewports/fixed_aspect_ratio_viewport.dart

```diff
@@ -37,6 +37,7 @@ class FixedAspectRatioViewport extends Viewport {
     _handleResize(size);
   }
 
+  @protected
```

Comment:
Wait, why is this needed? This is already private

Reply:
Haha, just something that was left in-between all the refactors, removing it.